### PR TITLE
Update README for Run MATLAB Tests

### PR DESCRIPTION
**Pull request by mw-hrastega**
Updated the text and examples. When reviewing the examples, please pay attention to the text in the YAML scripts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,72 @@
...
+      - name: Run tests and generate artifacts
         uses: matlab-actions/run-tests@v0
         with:
-            test-results-junit: test-results/results.xml
-            code-coverage-cobertura: code-coverage/coverage.xml
+          test-results-junit: test-results/results.xml
+          code-coverage-cobertura: code-coverage/coverage.xml
 ```
+## Run MATLAB Tests
+When you define your workflow in the `.github/workflows` directory of your repositoy, you can specify the **Run MATLAB Tests** action using the `run-tests` key. The action accepts optional inputs.
```
Comment (jwpereira) [inline README.md:53]:
```suggestion
When you define your workflow in the `.github/workflows` directory of your repository, you can specify the **Run MATLAB Tests** action using the `run-tests` key. The action accepts optional inputs.
```

Comment (jwpereira) [inline README.md:53]: Same as `run-command`: it's unlikely that a user would use `run-tests` unqualified without our org/version 🤔

**Review by jwpereira — APPROVED** [review]
Looks great 😁

File: README.md (hunk 1 above)
Comment (mw-hrastega) [reply README.md:53]: Fixed.

File: README.md [hunk 2]
```diff
@@ -1,57 +1,72 @@
...
 
-Use this action to run your MATLAB and Simulink® tests and generate artifacts
-such as JUnit test results and Cobertura code coverage reports. To run the tests
-in your project, the Run MATLAB Tests action uses the first MATLAB version on
-the runner's system `PATH`.
+The [Run MATLAB Tests](#run-matlab-tests) GitHub&reg; action enables you to run MATLAB&reg; and Simulink&reg; tests and generate artifacts such as JUnit test results and Cobertura code coverage reports. You can run tests and generate artifacts on a [GitHub-hosted](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners) or [self-hosted](https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/about-self-hosted-runners) runner:
 
-**Note**: By running the code in this action, you will be executing third-party
-code that is licensed under separate terms.
+- If you want to use a self-hosted runner, you must set up a computer with MATLAB (R2013b or later) as your runner. The action uses the first MATLAB version on the runner's path.
```
Comment (mcafaro) [inline README.md:5]: "The action uses the first MATLAB version on the runner's **system** path."

File: README.md (hunk 1 above)
Comment (mw-hrastega) [reply README.md]: Thanks, Johan. Does the updated text help?

**Review by mcafaro — APPROVED** [review]
Just a minor suggestion

File: README.md (hunk 2 above)
Comment (mw-hrastega) [reply README.md]: Good suggestion. Done!

**Pull request merged**